core/src/hash_table/entry.rs

```diff
@@ -14,7 +15,7 @@ use std::{
 /// data is stored as a JSON string
 #[derive(Clone, Debug, Serialize, Deserialize)]
 pub struct Entry {
```

Comment:
We just agreed on using the newtype pattern here and change this to
```pub struct Entry(String);```
https://doc.rust-lang.org/1.0.0/style/features/types/newtype.html